### ISSUE TITLE
SSL Error running librarian puppet 

### ISSUE THREAD

**Issue by neillturner** [neillturner/ec2dream#3]
when using test-kitchen with puppet and when running librarian puppet: 

if get error: 
Class: Faraday::SSLError Message: SSL_connect returned=1 errno=0 state=SSLv3 read server certificate B: certificate verify failed

see http://stackoverflow.com/questions/4528101/ssl-connect-returned-1-errno-0-state-sslv3-read-server-certificate-b-certificat
and for windows client see https://gist.github.com/fnichol/867550

basically need to download a cacert.pem file 
and before running ec2dream 
set SSL_CERT_FILE=C:/directoryoffile/cacert.pem
